site/content/en/docs/Reference/Commands/generate-docs.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
This doc should be hidden because the `generate-docs` command is hidden.

Reply:
done, thanks